pkg/collectors/flash_system.go

```diff
@@ -104,17 +105,69 @@ func (f *PerfCollector) Collect(ch chan<- prometheus.Metric) {
 	f.systems = updatedSystems
 
 	for systemName, fsRestClient := range f.systems {
+		var PoolsInfoList []PoolInfo
+		pools, mDisksList, err := getPoolAndMdisks(fsRestClient)
+		if err != nil {
+			log.Errorf("get pools or mdisks failed: %v", err)
+		}
+		for _, pool := range pools {
+			poolinfo := PoolInfo{}
+			poolinfo.PoolName = pool[MdiskNameKey].(string)
+			poolinfo.PoolMDiskList, err = getMDisksForPool(fsRestClient, poolinfo.PoolName, mDisksList)
+			if err != nil {
+				log.Errorf("get mdisks for pool failed: %v", err)
+			}
```

Comment:
I see we continue with the collection even when encountering a problem with getting the mdisks data, is that OK? it can cause inaccurate capacity stats, no?